soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -35,12 +36,12 @@ enum MatchSituation {
     in_play,         // normal play
 };
 
-enum Positions { Goalie, Defense, Offense, PenaltyPlayer, GoalKicker };
+enum Positions { Goalie, Defense, Offense, Runner, PenaltyPlayer, GoalKicker };
```

Comment:
you updated this enum, but didn't correctly update the switch-case in AAC that corresponds to this enum; so all of the positions assigned by coach are offset now. I see that you attempted to but commented it out. The correct behavior would have been to chance `case 3:` to be Runner, and change the old case 3 to 4, and 4 to 5.